src/sql/src/plan/statement/show.rs

```diff
@@ -456,6 +456,10 @@ pub fn show_indexes<'a>(
 ) -> Result<ShowSelect<'a>, PlanError> {
     let mut query_filter = Vec::new();
 
+    if on_object.is_none() && from_schema.is_none() {
+        query_filter.push("on_id NOT LIKE 's%'".into());
```

Comment:
This was/is needed because of the indexes you added recently, and by default we don't want to show those? What's the `SHOW` invocation if you want to show them?

Reply:
Actually by default we don't want to show any system indexes, on if `ON` or `FROM` is included. If we wanted to see one of the default indexes we would do something like
`SHOW INDEXES ON mz_types`
or
`SHOW INDEXES FROM mz_internal`